LICENSE

```diff
@@ -0,0 +1,403 @@
+Copyright (c) 2020, the SORSE20 committee.
+
+The contents of the SORSE20 website and it's source code repository are licensed
+under the CC BY 4.0 license (see below). The original software for creating this
+website, the Minimal Mistakes Jekyll Theme, is distributed under the MIT
+license, see https://github.com/mmistakes/minimal-mistakes#license, which also
+applies for the _includes and _layouts folder in the source code repository.
```

Comment:
```suggestion
See README.md on how and where to apply this license.
```

> For this reason so that GitHub recognizes it as CC-BY. 

alright, I just link to the README.md in LICENSE. This works fine

> But I can also do it as follow-up PR.

you can also edit the files directly here @bast, e.g. using the `Files changed` tab at the top and click the 
![image](https://user-images.githubusercontent.com/9960249/85209345-6bb7c400-b337-11ea-84ff-ed1edb995756.png) to make a suggestion (this is what I did here). Then we do not need another PR (see the [github docs](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)).